src/Mod/Sketcher/App/SketchObject.cpp

```diff
@@ -7576,17 +7572,7 @@ int SketchObject::carbonCopy(App::DocumentObject* pObj, bool construction)
         }
     }
 
-    // We shall solve in all cases, because recompute may fail, and leave the

```

Comment:
Perhaps we should leave this comment on. I am also seeing that there are cases where a `solve` is required without qualification (see split) in order for multiple tests to pass.